spec/models/user_spec.rb

```diff
@@ -11,6 +11,10 @@
     }
   end
 
+  before(:each) do
+    User.create!(valid_attributes)
```

Comment:
May be we can write it as:
```suggestion
    User.create!(user_attributes)
```
as we are passing invalid attributes also!